app/src/main/java/fr/free/nrw/commons/customselector/ui/selector/CustomSelectorActivity.kt

```diff
@@ -74,12 +77,41 @@ class CustomSelectorActivity: BaseActivity(), FolderClickListener, ImageSelectLi
         }
 
         // Open folder if saved in prefs.
-        if(prefs.contains(FOLDER_ID)){
+        if(prefs.contains(FOLDER_ID)) {
             val lastOpenFolderId: Long = prefs.getLong(FOLDER_ID, 0L)
             val lastOpenFolderName: String? = prefs.getString(FOLDER_NAME, null)
             val lastItemId: Long = prefs.getLong(ITEM_ID, 0)
             lastOpenFolderName?.let { onFolderClick(lastOpenFolderId, it, lastItemId) }
         }
+
+            if(isImageGalleryEmpty())
```

Comment:
Add a comment for this if statement, similar to the one above. 
Indentation.

Reply:
done